Solutions/Corvus.Testing.AzureFunctions.ReqnRoll/Corvus/Testing/AzureFunctions/ReqnRoll/FunctionsBindings.cs

```diff
@@ -0,0 +1,116 @@
+                context.Set(controller);
+            }
+
+            return controller;
+        }
+
+        /// <summary>
+        /// Retrieves the <see cref="FunctionConfiguration"/> from the context.
+        /// </summary>
+        /// <param name="context">The context in which the configuration is stored.</param>
+        /// <returns>The <see cref="FunctionConfiguration"/>.</returns>
+        /// <remarks>
+        /// If a <see cref="FunctionConfiguration"/> hasn't already been added to the context,
+        /// this method will create and add a new instance.
+        /// </remarks>
+        public static FunctionConfiguration GetFunctionConfiguration(ReqnrollContext context)
+        {
+            if (!context.TryGetValue(out FunctionConfiguration value))
+            {
+                value = new FunctionConfiguration();
+                context.Set(value);
+            }
+
+            return value;
+        }
+
+        /// <summary>
+        /// Start a functions instance.
+        /// </summary>
+        /// <param name="path">The location of the functions project.</param>
+        /// <param name="port">The port on which to start the functions instance.</param>
+        /// <returns>A task that completes once the function instance has started.</returns>
+        [Given(@"I start a functions instance for the local project '([^']*)' on port (\d*)")]
+        public Task StartAFunctionsInstance(string path, int port)
+        {
+            return GetFunctionsController(this.scenarioContext)
+                  .StartFunctionsInstanceAsync(path, port, "net8.0");
+        }
+
+        /// <summary>
+        ///     Start a functions instance.
+        /// </summary>
+        /// <param name="path">The location of the functions project.</param>
+        /// <param name="port">The port on which to start the functions instance.</param>
+        /// <param name="runtime">The id of the runtime to use.</param>
+        /// <returns>A task that completes once the function instance has started.</returns>
+        [Given(@"I start a functions instance for the local project '([^']*)' on port (\d*) with runtime '([^']*)'")]
+        public Task StartAFunctionsInstance(string path, int port, string runtime)
+        {
+            FunctionConfiguration configuration = FunctionsBindings.GetFunctionConfiguration(this.scenarioContext);
+            return GetFunctionsController(this.scenarioContext)
+                  .StartFunctionsInstanceAsync(path, port, runtime, "csharp", configuration);
+        }
+
+        /// <summary>
+        ///     Tear down the running functions instances for the scenario.
+        /// </summary>
+        /// <returns>Returns a Task for the operation.</returns>
+        [AfterScenario]
+        public async Task TeardownFunctionsAfterScenario()
+        {
+            await GetFunctionsController(this.scenarioContext).TeardownFunctionsAsync().ConfigureAwait(false);
```

Comment:
The SpecFlow version of this uses the `RunAndStoreExceptions` extension from `Corvus.Testing.SpecFlow` to ensure that any exceptions thrown while attempting to tear down the functions process don't prevent other After hooks from running. What was the reason for not including it here?